packages/outline-storybook/stories/components/outline-core-link.stories.ts

```diff
@@ -34,39 +41,41 @@ export default {
   },
   args: {
     linkHref: 'https://outline.phase2tech.com',
-    linkRel: 'external',
-    linkTarget: '_blank',
+    linkTarget: '',
     linkText: 'Sample Link',
   },
   parameters: {
     docs: {
       description: {
         component: `
-The \`outline-link\` component allows complete flexibility in how you choose to render a link element.
+The \`outline-core-link\` component allows complete flexibility in how you choose to render a link element.
 
 ## Description & Usage
 
-- Providing both a \`link-href\` and \`link-text\` attribute negates the need or usage of any slotted content.
-- Providing a \`link-href\` without the \`link-text\` attribute will use the slotted content as the content/text of the link.
-- Providing neither the \`link-href\` nor the \`link-text\` attributes will assumes the entire link, including \`<a></a>\` be passed into the slot from the consumer application. 
+- You must provide and set the \`link-href\` property to a valid URL.
+- You can optionally set the \`link-target\` property to a valid target value.
+- You can optionally set the \`link-text\` property to a valid text value.
+- You can optionally set the \`link-rel\` property to a valid rel value.
+- If the link is external, the component will automatically add \`rel="noopener noreferrer"\` to the link and open the link in a new tab. 
+- If you would like to put a link in a slot, you should now create a new component to handle this. 
```

Comment:
@bardleb @shaal @glitchgirl 
I don't like this wording in the docs: 

If you would like to put a link in a slot, you should now create a new component to handle this.

The render function demonstrates you CAN still put a link in the slot (most default fallback behavior).

```typescript
render(): TemplateResult {
  return html`
  <a
    href=${this.linkHref}
    rel="${ifDefined(this.linkRelRender())}"
    target="${ifDefined(this.linkTargetRender())}"
  >
    ${this.linkText ? html`${this.linkText}` : html`<slot></slot>`}
  </a>`;
}
```

Reply:
Nevermind, I think I'm being thick here. I was reading the code wrong.